reporters/http.go

```diff
@@ -103,14 +106,16 @@ func MakeHTTP(config *viper.Viper, logger *logrus.Logger, r *Reporters) {
 func loadDefaultHTTPConfigs(c *viper.Viper) {
 	c.SetDefault("reporters.http.putURL", "http://localhost:8080")
 	c.SetDefault("reporters.http.region", "test")
+	c.SetDefault("reporters.http.timeoutMs", "5000")
 }
 
 // NewHTTP creates an HTTP struct using putURL and region from config
 func NewHTTP(config *viper.Viper, logger *logrus.Logger) (*HTTP, error) {
 	loadDefaultHTTPConfigs(config)
 	putURL := config.GetString("reporters.http.putURL")
 	region := config.GetString("reporters.http.region")
-	client := NewHTTPClient(putURL)
+	timeoutMs := config.GetInt64("reporters.http.timeoutMs")
```

Comment:
on config/local.yaml you should save, for example, as `5s` and use here `config.GetDuration`. So no need to convert to time.Duration in line 118